app/src/main/java/de/westnordost/streetcomplete/data/meta/OsmTaggings.kt

```diff
@@ -31,3 +31,19 @@ val MAXSPEED_TYPE_KEYS = setOf(
 )
 
 const val SURVEY_MARK_KEY = "check_date"
+
+val KEYS_THAT_SHOULD_NOT_BE_REMOVED_WHEN_SHOP_IS_REPLACED = listOf(
+    "landuse", "historic",
+    // building/simple 3d building mapping
+    "building", "man_made", "building:.*", "roof:.*",
+    // any address
+    "addr:.*",
+    // shop can at the same time be an outline in indoor mapping
+    "level", "level:ref", "indoor", "room",
+    // geometry
+    "layer", "ele", "height", "area", "is_in",
```

Comment:
What about prefixed is_in? https://taginfo.openstreetmap.org//search?q=is_in%3A

Also, hertiage tags https://taginfo.openstreetmap.org/search?q=heritage

Reply:
I've never seen "is_in" used on anything that's not an admin or place object, but heritage, sure

Reply:
@TurnrDev [taginfo](https://taginfo.openstreetmap.org/keys/shop#combinations) does find few thousand cases of `is_in` in combination with `shop=*`, though (and few dozen thousands in combination with `amenity=*` etc.)

Reply:
@mnalis Surely adds:* tags are better suited in this case?

Reply:
@TurnrDev  Yes, likely `addr:*` are better suited (although I'm not sure how `is_in` in [6815996361](http://www.openstreetmap.org/node/6815996361) or [6153771467](http://www.openstreetmap.org/node/6153771467) should be mapped correctly, for example); but that doesn't change the fact that those tags are there; and creating AI which would rename them to appropriate `addr:*` tags is likely much more complex then just leaving them as they are...